Early Access for workspace-classes and usage-based billing (#2715)
gitpod-io/website: gitpod/docs/usage-based-billing.md

# {title}

> > {title} is currently in [Early Access](/docs/references/gitpod-releases) Please [contact us](/contact/support) for an invitation, or [send feedback](https://github.com/gitpod-io/gitpod/issues/12636).

Reviewer: **question**: Maybe unnecessary, but what do you think of rephrasing this so that we lower the expectations on getting an invitation? Otherwise, this could be misinterpreted by users. Plus, minor missing period.

```suggestion
> {title} is currently in [Early Access](/docs/references/gitpod-releases). Please [contact us](/contact/support) to request access, or [send feedback](https://github.com/gitpod-io/gitpod/issues/12636).
```

Usage-based billing provides a flexible way for teams and individual users to pay for Gitpod.

Instead of pre-paying for yourself or for each team member every monthb, Gitpod will charge for usage based on how long each workspace runs, and its allocated [resources](/docs/workspace-classes).

With usage-based billing you will be charged only for the usage incurred during a billing period.

## Credits

Gitpod usage is measured in **credits**. Your invoice will show the total amount of credits consumed in a billing period.

Larger [workspace classes](/docs/workspace-classes) will use credits at a faster rate than smaller workspace classes. E.g. `Standard` workspaces use `10 credits per hour`, whereas `Large` workspaces use `20 credits per hour`.

Users or teams who have been granted free credits, or who have purchased credits in advance, will use those credits first, and only see charges for additional usage above that.

> 

Reviewer: **issue(non-blocking)**: Just cross-posting a disagreeing perspective from https://github.com/gitpod-io/website/pull/2657#discussion_r961697701 on including images in our docs as this work-in-progress already includes obsolete screenshots below—see the missing currency selector on the payment modal.

No action required. Moving forward with the images is also fine. Your call. 🏀

Author: thanks.

> 

Reviewer: **issue(non-blocking)**: Posting this here as the closest point in code to add this. Using the _Gitpod Usage-Based Billing_ in the screenshots as an example team, besides the minor casing mismatch with the rest of the docs, could be confusing for users as they might expect to see this element fixed in their dashboard. I'd suggest to use another example team like _ACME Team_ or a random term to help users associate this element with their team.

Moving forward with the screenshots as they are is also fine. Your call. 🏀

Author: good feedback - thanks.
